### ISSUE TITLE
Polyaxon does not enable a disabled node after removing and readding again

### ISSUE THREAD

**Issue by mouradmourafiq** [polyaxon/polyaxon#190]
### Describe the bug

> I removed and added again the same node in the k8s cluster (first `kubeadm reset` and then `kubeadm join` on that node), the corresponding pods were created correctly, but in the database flag `Is_current` was left to `False`, so I have manually changed it to `True`.
> 
> Maybe I didn't wait enough to change it automatically to `True` ?

**Comment by ajuric** [comment]
If it helps, just to let you know:

I have tested this again. I have removed one node from the cluster (on the master: `kubectl drain <NODE_NAME> --delete-local-data --force --ignore-daemonsets` and `kubectl delete nodes <NODE_NAME>`, on the node: `kubeadm reset`), added it back again (on the node `kubeadm join ...`) and waited for more than one hour but Polyaxon didn't recognize it. So I had to manually change the `Is_current` flag to true.
